recipes/leftbar/general_row/general_row_constants.js

```diff
@@ -0,0 +1,33 @@
+export const LEFTBAR_GENERAL_ROW_TYPES = {
+  INBOX: 'inbox',
+  CONTACTS: 'contacts',
+  CHANNELS: 'channels',
+  THREADS: 'threads',
+  LOCKED_CHANNEL: 'locked channel',
+  CONTACT_CENTER: 'contact center',
+  DIALBOT: 'dialbot',
+};
+
+export const LEFTBAR_GENERAL_ROW_ICON_MAPPING = {
+  [LEFTBAR_GENERAL_ROW_TYPES.INBOX]: 'inbox',
+  [LEFTBAR_GENERAL_ROW_TYPES.CONTACTS]: 'contacts',
+  [LEFTBAR_GENERAL_ROW_TYPES.CHANNELS]: 'hash',
+  [LEFTBAR_GENERAL_ROW_TYPES.THREADS]: 'thread',
+  [LEFTBAR_GENERAL_ROW_TYPES.LOCKED_CHANNEL]: 'lock',
+  [LEFTBAR_GENERAL_ROW_TYPES.CONTACT_CENTER]: 'stop-filled',
```

Comment:
[The Figma file](https://www.figma.com/file/oTY56LgQha8ljb4c8w4GtO/DT-Shared%3A-App-Components?node-id=9240%3A7131&t=fUu3sW7wEstPCHg4-11) didn't use an icon, and this might be brittle in the future if the `stop-filled` icon is redrawn to something that doesn't fit this use case.

We may draw an icon that is very specifically a "CC" icon and can be colorized. In the meantime, let's make it a raw square html element that is...

```
width: var(--icon-size-200);
height: var(--icon-size-200);
border-radius: var(--size-200);
```

Reply:
Hm yes we probably need a dedicated icon for the CC. Updated to use HTML for now https://github.com/dialpad/dialtone-vue/pull/769/commits/3679226bf2cf3c56001411cf4b2d48e1d577b0bb